components/layout/ContextBar.tsx

```
@@ -0,0 +1,56 @@
+
+export const ContextBar: React.FC<ContextBarProps> = ({ sections = DEFAULT_SECTIONS, className }) => {
+  const ids = sections.map(s => s.id)
+  // Assumption: existing hook signature only accepts { ids }
+  const active = useScrollSpy({ ids })
+  const { track } = useAnalytics()
+  const router = useRouter()
+
+  if(!router.query.walletId) return null
+
+  return (
+    <div className={cn('sticky top-14 z-30 w-full border-b border-[var(--glass-border-strong)] bg-[var(--glass-bg)]/65 backdrop-blur-xl', className)} role="navigation" aria-label="In-page">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)